gm4_shiny_items/data/gm4_shiny_items/functions/blacklist_items.mcfunction

```diff
@@ -0,0 +1,125 @@
+# Helmets (6)
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:turtle_helmet"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:leather_helmet"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:chainmail_helmet"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:iron_helmet"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:diamond_helmet"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:golden_helmet"}}] run tag @s add gm4_shiny_blacklist
+# Chestplates (5)
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:elytra"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:leather_chestplate"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:golden_chestplate"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:chainmail_chestplate"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:iron_chestplate"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:diamond_chestplate"}}] run tag @s add gm4_shiny_blacklist
+# Leggings (5)
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:leather_leggings"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:chainmail_leggings"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:iron_leggings"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:diamond_leggings"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:golden_leggings"}}] run tag @s add gm4_shiny_blacklist
+# Boots (5)
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:leather_boots"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:chainmail_boots"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:iron_boots"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:diamond_boots"}}] run tag @s add gm4_shiny_blacklist
+execute if entity @s[tag=!gm4_shiny_blacklist,nbt={Item:{id:"minecraft:golden_boots"}}] run tag @s add gm4_shiny_blacklist
+
+### ITEMS THAT CANT GET THE ENCHANTMENT GLOW ###
```

Comment:
player heads can't get the enchantment glow, possibly also applies to all heads/skulls